packages/server-web/src/client/component/TodoItem.vue

```diff
@@ -71,13 +71,16 @@ export default {
     },
     isDragging() {
       return this.$store.getters.isDragging;
+    },
+    isEditing() {
+      const { todo } = this.$props;
+      return this.$store.getters.isEditing[todo.todoId];
```

Comment:
do we need to have this as global isEditting? isn't it enough for the particular item to know, that it's in editing mode?